dspace-server-webapp/src/test/resources/org/dspace/app/rest/openalex-institution-multiple.json

```diff
@@ -0,0 +1,5575 @@
+{
```

Comment:
This is a rather large test file (over 5,000 lines).  Would it be possible to trim this down to *only* the content we need for testing?  It just seems like a very large file for us to store just for testing.